api/v1/cosmosfullnode_types.go

```diff
@@ -84,18 +84,6 @@ type FullNodeSpec struct {
 	// Used for debugging.
 	// +optional
 	InstanceOverrides map[string]InstanceOverridesSpec `json:"instanceOverrides"`
-
-	// VolumeSnapshot creates recurring VolumeSnapshots of a PVC.
-	// A VolumeSnapshot is a CRD (installed in GKE by default).
-	// See: https://kubernetes.io/docs/concepts/storage/volume-snapshots/
-	// This enables recurring, consistent backups.
-	// To prevent data corruption, the pod is temporarily deleted while the snapshot takes place which could take
-	// several minutes.
-	// Therefore, if you enable VolumeSnapshots, you must use replica count >= 2 to prevent downtime.
-	// Only 1 VolumeSnapshot is created at a time, so at most only 1 pod is temporarily deleted.
-	// Multiple, parallel VolumeSnapshots are not supported.
-	// +optional
-	VolumeSnapshot *VolumeSnapshotSpec `json:"volumeSnapshot"`
```

Comment:
This was a mistake. The initial spike of creating recurring snapshots initially was part of CosmosFullNode before it was broken out into its own CRD/controller. 😬 This has no bearing on functionality.